tests/itests/pom.xml

```diff
@@ -48,14 +48,6 @@
             <type>xml</type>
             <scope>test</scope>
         </dependency>
-        <dependency>
-            <groupId>ddf.security</groupId>
-            <artifactId>security-services-app</artifactId>
```

Comment:
Is this the only dead one? What about `platform-app` and `admin-app`?

Reply:
Good catch, yeah, the platform-app needs to be removed. The admin-app still exists, it'll need to be updated once https://github.com/codice/ddf/pull/3300 goes in